libsolutil/CDCL.cpp

```diff
@@ -230,9 +245,40 @@ std::pair<Clause, size_t> CDCL::analyze(Clause _conflictClause)
 	return {move(learntClause), backtrackLevel};
 }
 
-void CDCL::addClause(Clause _clause)
+void CDCL::addClause(const Clause& _lits)
 {
-	m_clauses.push_back(make_unique<Clause>(move(_clause)));
+	if (!ok) return;
+
+	Clause clause{_lits};
+	Clause clause_updated;
```

Comment:
```suggestion
	Clause updatedClause;
```

Reply:
Thanks, I see, now I understand the style better :)